backend/data/download-images.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+import fetch from 'node-fetch';
+import { ProfileImage } from '../src/DataTypes';
+
+const fs = require('fs-extra');
```

Comment:
Change this to use import and `yarn workspace backend add fs-extra`

Reply:
done!

Reply:
![image](https://user-images.githubusercontent.com/65922473/113804637-f8980680-972c-11eb-9aaf-358190392574.png)
There's still another linting error that needs to be fixed